tests/Resources/ti.contacts.group.test.js

```diff
@@ -8,45 +8,42 @@
 /* eslint-env mocha */
 /* eslint no-unused-expressions: "off" */
 'use strict';
-var should = require('./utilities/assertions'),
-	utilities = require('./utilities/utilities');
+const should = require('./utilities/assertions');
+const utilities = require('./utilities/utilities');
```

Comment:
<!--
  0 failure: 
  1 warning:  tests/Resources/t...
  
  
  DangerID: danger-id-Danger;
  File: tests/Resources/ti.contacts.group.test.js;
  Line: 12;
-->

- :warning: tests/Resources/ti.contacts.group.test.js line 12 – 'utilities' is assigned a value but never used. (no-unused-vars)